docs/competition/team-support/battery-charging.md

```diff
@@ -7,44 +7,76 @@ original:
- * Red insulating tape
- * Example good batteries (one brand new, one squidgy)
+### Equipment
+
+There are two locations involved in battery charging:
+
+- Helpdesk
+    - 1 x Discharged Battery Box (18L RUB), clearly labelled
+    - 1 x Charged Battery Box (18L RUB), clearly labelled
+    - 1 x LiPo Bag for damaged batteries, clearly labelled, red taped
+    - At least one roll of red insulating tape
+- Battery Charging Station
+    - At least `n` battery chargers
+    - Enough LiPo charging bags for every charger.
+    - 1 x Discharged Battery Box (18L RUB), clearly labelled
+    - 1 x Charged Battery Box (18L RUB), clearly labelled
+    - 1 x Damaged battery box (18L RUB), clearly labelled, red taped
+    - At least one roll of red insulating tape
+
+
+### Safe Use of Battery Charging Equipment - Battery Charging Station 
+
+- There should be `0.5A` of mains electricity available per charger.
+- It should be ensured that any power distribution / 4 ways is setup properly and able to handle the current draw.
+- Additionally, there should be no trailing wires across the floor to prevent charging batteries falling on the floor.
 
 ## Volunteer Requirements
 
-Every volunteer performing the Battery Charging role must have read this document fully. Any doubts must be discussed with Rich Barlow ASAP.
+Every volunteer performing the Battery Charging role must have read this document fully. Any doubts must be raised before beginning the role.
 
 ## Safety
 
 The biggest risk of damage or injury is from the banana plugs coming unplugged from the charger. These are the red and black plugs that go into the side of the charger. All of the chargers should have the cables cable-tied to the charger to make it difficult to remove these plugs. Under no circumstances should these red and black plugs be unplugged from the chargers.
 
 The batteries themselves have many built in safety features and, unless severely abused, are completely safe. All batteries undergoing charging must be placed inside a fireproof charging bag. In the extremely unlikely event of a fire due to a battery DO NOT attempt to tackle the fire - the batteries contain a flammable metal (Lithium) which cannot be safely tackled with most fire extinguishers.
 
-If at any point you have any doubts about safety, immediately summon the Competition Team Support Coordinator (Rich Barlow) and the Health & Safety Coordinator (Scarzy) via [Helpdesk](/competition/team-support/helpdesk).
+If at any point you have any doubts about safety, immediately ask the helpdesk to summon the Health & Safety officer.
 
 ## Procedures
 
-### Swapping a team's battery
+### Swapping a team's battery - Helpdesk
+
+- Teams will bring their discharged battery to the Helpdesk
+- Discharged batteries should be placed in the discharged battery box
+    - If the battery is damaged, red tape should be placed on it and it should be placed in the damaged LiPo bag instead of the discharged battery box. 
```

Comment:
This means that volunteers on _both_ helpdesk and battery charging (which may be in different locations) need training in what damaged batteries look/feel like. What benefit does this serve over letting that be done only at the battery charging station?

Reply:
Wouldn't it be wise for both sites to be checking for battery damage? I feel that it's better to be safe than sorry in this case.

Reply:
Possibly, however the overhead in training is non-trivial. There is also the possible outcome that each assumes the other has done a good enough check and then doesn't check again.

In my experience shared responsibilities are ok if all parties are co-located and constantly working closely with each other, but otherwise result in everyone thinking that the job anyone could do would be done by someone else. I'd therefore encourage we make this a specific responsibility of the battery-charging station.

I'd be happy for @Scarzy to overrule me here though if he wanted.

Reply:
> I'd therefore encourage we make this a specific responsibility of the battery-charging station.

I'd agree with this. It is the responsibility of battery charging to do the check., otherwise we run the risk of some being overlooked.

However, that does not mean that helpdesk shouldn't apply common sense, if a battery turns up emitting smoke I would be rather disappointed if they just put it in the normal box and ignored it...

I'm fairly sure in previous years that the batteries have _always_ been stored in the fireproof bags. Helpdesk put them in a bag when received from a team, transport them to battery charging in said bag, and after charging the batteries get returned to and stored at helpdesk in a bag. In which case it becomes a much smaller risk, and a bit moot for helpdesk to do any checking anyway - the handling procedure would be the same.